cookie_monster/templates/cookie_monster/base.html

```diff
@@ -82,6 +82,7 @@
             onAccepted: onAccepted,
             cookieId: '{{ cookie_monster.cookie.identifier }}',
             cookieAge: '{{ cookie_monster.cookie.age }}',
+            cookieSameSite: '{{ cookie_monster.cookie.samesite }}',
```

Comment:
Probably it's worth considering the use of the cookie options object instead of the single properties.